ipatests/test_integration/test_epn.py

```diff
@@ -59,6 +62,79 @@ def postconf(host, option):
     host.run_command(r"postconf -e '%s'" % option)
 
 
+def generate_user(host, uid, **kwargs):
+    user_defaults = {
+        'objectClass': [
+            'ipaobject',
+            'person',
+            'top',
+            'ipasshuser',
+            'inetorgperson',
+            'organizationalperson',
+            'krbticketpolicyaux',
+            'krbprincipalaux',
+            'inetuser',
+            'posixaccount',
+            'ipaSshGroupOfPubKeys',
+            'mepOriginEntry'],
```

Comment:
Not a deal breaker but I wonder if the list of objectclasses can be obtained via ipa config-show. This would future-proof the test.

Reply:
It would be probably better. I guess I could leave it as it is right now and add comment for future revisions